WeakAurasOptions/AuthorOptions.lua

```diff
@@ -704,6 +705,178 @@ local typeControlAdders = {
     widthOption.order = order
     order = order + 1
 
+    return order
+  end,
+  multiselect = function(option, args, data, order, i)
+    args["option" .. i .. "width"] = nil
+    local values
+    local conflict = {} -- magic value
+    if option.references then
+      values = {}
+      for childID, optionID in pairs(option.references) do
+        local childData = data[childID]
+        local childValues = childData.authorOptions[optionID].values
+        local i = 1
+        while i <= #values or i <= #childValues do
+          local value = values[i]
+          if value == conflict then
+            -- conflicts can't ever be resolved at this point
+          elseif value == nil then
+            -- set the new value
+            values[i] = childValues[i]
+          elseif value ~= childValues[i] then
+            -- either this child has a conflicting value at this index
+            -- or it's already ended and is nil, so we need to mark a conflict
+            values[i] = conflict
+          end
+          i = i + 1
+        end
+      end
+    else
+      values = option.values
+    end
+    local defaultValues = {}
+    for i, v in ipairs(values) do
+      if v == conflict then
+        defaultValues[i] = "|cFF4080FF" .. L["Value %i"]:format(i)
+      else
+        defaultValues[i] = v
+      end
+    end
+    args["option" .. i .. "default"] = {
+      type = "multiselect",
+      width = WeakAuras.normalWidth,
+      name = L["Default"],
+      -- name = name(data, option, "default", L["Default"]),
```

Comment:
remove?

Reply:
Ah, I forgot about that. I was lazy and didn't want to deal with figuring out how to blue the name, and forgot to fix that.

Reply:
But, on further thought, the sub-values do the trick i think. So I'll just remove the commented code.